### PR TITLE
Add ipynb to try Synbols

### PR DESCRIPTION
**Pull request by Dref360**
# Summary

Add an ipynb that shows how to use Synbols and how to train something with it (using Pytorch Lightning).

Questions:
* Do we want this ipynb?
* Do we add a new step in the docker to install pytorch_lightning/jupyter
* Do we make a new cli command to launch this notebook :)

### CODE REVIEW

**Comment by aldro61** [comment]
Awesome! 🥇 

Maybe it's worth adding a comment about how people should either run that notebook inside the docker image or copy the code to files and use the `synbols` command to run it.

**Comment by Dref360** [comment]
I edited the readme and added a command to the makefile.

**Comment by aldro61** [comment]
Installing synbols inside the docker image will likely break the `synbols` command. The reason is that the user's system installation of synbols is injected into the image at runtime. What I would suggest is to add an endpoint to the setup.py. I'll try to make an example of what I mean.

**Comment by aldro61** [comment]
Also, it creates a chicken and egg problem since the docker needs to be built before we release the new version to pypi.

**Comment by aldro61** [comment]
I pushed a proposal @Dref360. Let me know what you think. The only thing left to do would be to add a port argument to the `synbols` command and to include it in the call to `docker run`.

**Comment by aldro61** [comment]
That command would allow users to run any jupyter notebook with their installation of synbols available.

**Comment by aldro61** [comment]
I had to downgrade some packages to really old versions to get jupyter to work. I see many examples where people get up and running by just `pip install jupyterlab`. Maybe we would be able to do the same. If this works, I don't see the point of limiting the installation to jupyter notebooks only.

File: docker/Dockerfile [hunk 1]
```diff
@@ -83,9 +83,24 @@ RUN python -c "import matplotlib.pyplot as plt"
...
+
+# ------------------------------------------------------------------------
+# Notebook examples
+# ------------------------------------------------------------------------
+RUN pip install jupyter matplotlib pytorch_lightning torch torchvision
+RUN pip install --upgrade ipython Pygments>=2.6.1
+RUN mkdir /.local && chmod -R 777 /.local
+
+# Copy examples to image
+COPY ./examples/ /examples
```
Comment (aldro61) [inline docker/Dockerfile:96]: It's not necessary to copy the examples to the image, but it doesn't hurt.

**Review by aldro61 — APPROVED** [review]
LGTM. Ready to merge.

**Pull request merged**